### DISCUSSION TITLE
Line within a group

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [fabricjs/fabric.js#7846] [Q&A] [answered]
Hey there,

I have a problem here in my example: https://jsfiddle.net/201flaviosilva/jvmL5bhf/

Basically I'm adding 3 lines to form a triangle, but supposedly at the 0x0 position of the group, but for some reason they are starting to be drawn in the centre of the group.

(Sorry if this is a stupid question, and also if I was confused, and thanks to everyone who answers (if anyone answers 🥲)) :)

**Comment by ShaMan123** [comment, marked as answer]
As mentioned previously in my answers, for an object the point (0,0) is the center of it's containing group

**Reply by 201flaviosilva** [reply]
hmmm ok, thanks
